pkg/testutil/alert_custom_validation_test.go

```diff
@@ -0,0 +1,115 @@
+package testutil
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/util/intstr"
+
+	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
```

Comment:
nit: join the imports

Reply:
WDYM?

Reply:
no empty line between them, both external imports

Reply:
Done